clouddriver-gae/src/main/groovy/com/netflix/spinnaker/clouddriver/gae/GaeCloudProvider.groovy

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Google App Engine (GAE) declaration as a {@link CloudProvider}.
+ */
+
+package com.netflix.spinnaker.clouddriver.gae
+
+import com.netflix.spinnaker.clouddriver.core.CloudProvider
+import java.lang.annotation.Annotation
+import org.springframework.stereotype.Component
+
+@Component
+class GaeCloudProvider implements CloudProvider {
+  public static final String GAE = 'gae'
```

Comment:
I think having that field is a mistake, naming a variable after a value is silly. It's brittle (changing the value means changing the variable name), and redundant (why have GAE and 'gae'). Why not change the other references to this field to be `GaeCloudProvider.id`?